data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java

```diff
@@ -67,14 +67,20 @@ public PhysicalParameter getPhysicalParameter() {
 
     @Override
     public Optional<Double> getMinThreshold(Unit requestedUnit) throws SynchronizationException {
-        // TODO : handle cases where direction != Direction.BOTH
-        return Optional.of(-convert(getAbsoluteMax(), unit, requestedUnit));
+        if (direction == Direction.OUT) {
```

Comment:
Do we really want to keep Direction.IN and Direction.OUT ?
I would prefer (and @terriervik too), Direction.ORIGIN_TO_EXTREMITY and Direction.EXTREMITY_TO_ORIGIN, or (but we are less fans) Direction.DIRECT and Direction.INDIRECT

Reply:
modified to Direction.DIRECT and Direction.OPPOSITE